### DISCUSSION TITLE
Is there a basic documentation outside of notion?

### DISCUSSION THREAD

**Discussion by TravelTrader** [jbakse/p5.party#74] [Q&A] [answered]
I’d like to read about the server installation shortly and without creating a notion account.

Is there a basic documentation outside of notion?

Or can you please explain shortly the server installation?
Is it done with node.js? Or a source build?

Thank you.

**Comment by jbakse** [comment, marked as answer]
The notion docs are public and don't require a notion account to view, BUT...

The links on p5party.org to the docs were wrong, leading to the login request. I've updated the link.

[The Main Docs Page](https://twisty-alder-868.notion.site/Welcome-to-p5-party-887564cad8ec455e9bee994362322f2e)

Thanks for bringing this to my attention!

**Reply by jbakse** [reply]
Also, p5.party uses a [deepstream](https://github.com/deepstreamIO/deepstream.io) server, without modification. So you might want to look at their docs also.

**Comment by TravelTrader** [comment]
@jbakse Thank you very much. The notion link didn’t open without login at least on iOS.

But I read about the Heroku solution. That’s great. I’ll definitely play around with it. Good approach!
